### PR TITLE
add full behaviour map and tests for unique outcomes

### PR DESCRIPTION
**Pull request by rossfauldsbjss**
MES-2068

Extend the service to contain a full mapping of all outcomes as defined in this spreadsheet

https://wiki.i-env.net/pages/viewpage.action?pageId=34602121

additionally create unit tests to test each unique scenario 
(a lot of the outcomes share the same settings)

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github

- N/A Screenshot(s) captured on the physical device (if applicable)

### CODE REVIEW

File: src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts [hunk 1]
```diff
@@ -159,4 +160,552 @@ describe('OutcomeBehaviourMapProvider', () => {
...
+  // 20  DONE
+  // 21  DONE
+  // 22 DONE
+  // 23 DONE
+  // 33 DONE
+  // 40 DONE
+  // 51 DONE
+  // 69 DONE
+
+  fdescribe('outcome testing', () => {
```
Comment (roberthall12) [inline src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts]: `fdescribe` left in

File: src/pages/office/office-behaviour-map.ts [hunk 2]
```diff
@@ -2,21 +2,693 @@ import { OutcomeBehaviourMapping } from '../../providers/outcome-behaviour-map/o
 
 export const behaviourMap: OutcomeBehaviourMapping = {
   ['1']: {
-    ['routeNumber']: { display: 'Y', defaultValue: null, showNotApplicable: false },
```
Comment (szabolcs-keresztes) [inline src/pages/office/office-behaviour-map.ts]: Why is this in the app?

File: src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts (hunk 1 above)
Comment (rossfauldsbjss) [reply src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts]: removed comments and fdescribe

File: src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts [hunk 3]
```diff
@@ -35,7 +36,7 @@ describe('OutcomeBehaviourMapProvider', () => {
     });
 
     outcomeBehaviourMapProvider = TestBed.get(OutcomeBehaviourMapProvider);
-    outcomeBehaviourMapProvider.setBehaviourMap(behaviourMap);
+    outcomeBehaviourMapProvider.setBehaviourMap(simpleBehaviourMap);
```
Comment (ammarhaiderbjss) [inline src/providers/outcome-behaviour-map/__tests__/outcome-behaviour-map.spec.ts]: Can't see where we're using this `setBehaviourMap`? We seem to be resetting it to use `behaviourMap` in all the tests?

**Pull request merged**